### PR TITLE
feat(bottom sheet): Adding Sticky header to the bottom sheet

### PR DESCRIPTION
**Pull request by dievazqu**
### Description

Adding Sticky header to the bottom sheet. Extracted logic from #3613

### CODE REVIEW

File: src/send/Send.tsx [hunk 1]
```diff
@@ -246,7 +251,8 @@ function Send({ route }: Props) {
         origin={TokenPickerOrigin.Send}
         onTokenSelected={onTokenSelected}
         onClose={closeCurrencyPicker}
-        tokens={isOutgoingPaymentRequest ? stableTokens : tokensWithBalance}
+        tokens={sortedTokens}
```
Comment (jeanregisser) [inline src/send/Send.tsx:254]: Looks like this tokens change should be done separately, right?

**Review by jeanregisser — COMMENTED** [review]
Thanks for splitting.

File: src/send/Send.tsx (hunk 1 above)
Comment (dievazqu) [reply src/send/Send.tsx:254]: What do you mean? This is not changing anything actually, we were sorting inside the TokenBottomSheet, now each component can decide how to sort that list...

Comment (jeanregisser) [reply src/send/Send.tsx:254]: Oh sorry I see! All good, I initially thought it was an unrelated change.

**Review by jeanregisser — APPROVED** [review]

**Pull request merged**